packages/elements/src/tree/helpers/renderer.ts

```diff
@@ -24,7 +24,7 @@ export class TreeRenderer extends Renderer {
     let currentMode: TreeManagerMode;
     let currentComposer: CollectionComposer<TreeDataItem>;
 
-    super((item: TreeDataItem, composer: CollectionComposer<TreeDataItem>, element = document.createElement('ef-tree-item')): HTMLElement => {
+    super((item: TreeDataItem, composer: CollectionComposer<TreeDataItem>, element: TreeItem = document.createElement('ef-tree-item')): HTMLElement => {
```

Comment:
You shouldn't need to add the type here.

Reply:
Right because createElement should already return correct type from `HTMLElementTagNameMap`. Fixed.